tests/null_queries/tests.py

```diff
@@ -57,28 +57,28 @@ def test_reverse_relations(self):
         obj = OuterA.objects.create()
         self.assertQuerysetEqual(
             OuterA.objects.filter(inner__third=None),
-            ['<OuterA: OuterA object>']
+            [repr(obj)]
```

Comment:
We're preferring to use `assertSequenceEqual(queryset, [obj])` rather than `assertQuerysetEqual()` -- could you make that change in a separate PR to keep this diff simpler?